test/test_util.dart

```diff
@@ -0,0 +1,400 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
```

Comment:
It's probably overkill to copy over the entire package:vm_service testing harness. Is there some subset you can pull out of here for your purposes?

Reply:
Done